CompassSDK/Storage/CompassStorage.swift

```diff
@@ -9,8 +9,8 @@ import Foundation
 
 protocol CompassStorage {
     func addVisit()
-    var suid: String {get}
-    var userId: String? {get set}
+    var suid: String? {get set}
```

Comment:
suid and userId were used around all the code in the other way around. Keys were swapped during api call serialization, really confusing